File: TCDNew-master/src/ui.android/lib/models/state/StateAuth.dart
import 'package:tiq/models/dtos/UserTokenResp.dart';

class StateAuth {
  UserTokenResp _userToken;

  StateAuth(UserTokenResp userToken) {
    if (userToken != null) {
      this._userToken = UserTokenResp(userToken.authToken, userToken.displayName);
    }
  }

  bool get isAuthorised => this._userToken != null;
  String get authToken => this._userToken.authToken;
  String get displayName => this._userToken.authToken;
}

File: TCDNew-master/src/ui.wince/Cen.Wms.Client/Models/State/StateAuth.cs
using Cen.Wms.Client.Models.Dtos;

namespace Cen.Wms.Client.Models.State
{
    class StateAuth
    {
        private readonly UserTokenResp _userToken;

        public StateAuth(UserTokenResp userToken)
        {
            if (userToken != null)
            {
                _userToken = new UserTokenResp { AuthToken = userToken.AuthToken, DisplayName = userToken.DisplayName };
            }
        }

        public bool IsAuthorised { get { return _userToken != null; } }
        public string AuthToken { get { return _userToken.AuthToken; } }
        public string DisplayName { get { return _userToken.DisplayName; } }
    }
}
